### ISSUE TITLE
Queries with multiview + wait causes spurious UNASSIGNED-CoreValidation-DrawState-InvalidQuery 

### ISSUE THREAD

**Issue by juston-li** [KhronosGroup/Vulkan-ValidationLayers#6107]
**Environment:**
 - OS: Linux
 - GPU: anv
 - SDK or header version if building from repo: 1.3.243
 - Options enabled (synchronization, best practices, etc.)

Per spec 
> If queries are used while executing a render pass instance that has
> multiview enabled, the query uses N consecutive query indices in the
> query pool (starting at query) where N is the number of bits set in
> the view mask in the subpass the query is used in.

When `vkCmdCopyQueryPoolResults` is used with `VK_QUERY_RESULT_WAIT_BIT` and multiview enabled, VVL logs these errors.

> Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidQuery ]
> Object 0: handle = 0x7b39e4902ed0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; | MessageID = 0x46ab3a6e |
> vkCmdCopyQueryPoolResults(): Requesting a copy from query to buffer on VkQueryPool 0xd897d90000000016[]
> query 1: waiting on a query that has been reset and not issued yet
> 
> Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidQuery ]
> Object 0: handle = 0x7b39e4902ed0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; | MessageID = 0x46ab3a6e |
> vkCmdCopyQueryPoolResults(): Requesting a copy from query to buffer on VkQueryPool 0xd897d90000000016[]
> query 2: waiting on a query that has been reset and not issued yet
> 
> Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidQuery ]
> Object 0: handle = 0x7b39e4902ed0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; | MessageID = 0x46ab3a6e |
> vkCmdCopyQueryPoolResults(): Requesting a copy from query to buffer on VkQueryPool 0xd897d90000000016[]
> query 3: waiting on a query that has been reset and not issued yet

In this instance, the BeginQuery was called on query 0 inside a renderpass with multiview enabled, generating 3 additional queries.

VVL thinks queries 1-3 have not been started because BeginQuery wasn't explicitly called for them.

**Expected behavior**
VVL should recognize multiview generates additional queries and include them when tracking Begin/End Query

**Additional context**
This was produced with venus+anv running  dEQP-VK.multiview.\*queries\* with https://gitlab.freedesktop.org/mesa/mesa/-/commit/e6cffa1f0e4e54616629177e09881784e33328db

There was some tracking added in render passes here: https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/4451/commits/73b10704bf98f9bfaa7d1d59925be4773beaa801

**Comment by zhangyiwei** [comment]
internal vk-gl-cts issue 4550 might help repro once something is up

**Comment by spencer-lunarg** [comment]
(a while ago now) the spec was updated and added `VUID-vkCmdResetQueryPool-None-02841` and `VUID-vkCmdCopyQueryPoolResults-None-07429` (while removing the `UNASSIGNED` VUs)

@juston-li can we get an update if this has fixed the issue, I can't recall if there was something actually left to do here

**Comment by spencer-lunarg** [comment]
Closing - we have the 2 new VUs added, if there are still issues with queries, please let me know, open a new issue and will be happy to take action on it
